dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -125,4 +127,16 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
             }
         }
     }
+
+    /**
+     * help text output
+     */
+    private void wesCommandSuggestions() {
+        String wesEntryType = abstractEntryClient.getEntryType().toString().toLowerCase();
+        out("Useful commands for working with WES runs:");
+        out("");
+        out("To get status for a run: dockstore " + wesEntryType + " wes status --id <run id> [--verbose][--wes-url <WES URL>][--wes-auth <auth>]");
+        out("To cancel a run: dockstore " + wesEntryType + " wes cancel --id <run id> [--wes-url <WES URL>][--wes-auth <auth>]");
```

Comment:
Same as previous comment -- put the actual id in there.

Reply:
Added the run id to the message